Formula/w/wownero.rb

```diff
@@ -45,27 +44,14 @@ class Wownero < Formula
   depends_on "libsodium"
   depends_on "libusb"
   depends_on "openssl@3"
-  depends_on "protobuf@21"
+  depends_on "protobuf"
```

Comment:
`brew linkage` isn't picking it up so looks like this doesn't support newer Protobuf yet. I wonder if the same issue applies to Monero and we didn't notice.

Reply:
I checked and `monero` is fine.
I reverted to protobuf@21